cmd/s2i/main.go

```diff
@@ -83,6 +83,13 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
 		Use:   "build <source> <image> [<tag>]",
 		Short: "Build a new image",
 		Long:  "Build a new Docker image named <tag> (if provided) from a source repository and base image.",
+		Example: `
+# Build an application Docker image from a Git repository
+$ s2i build git://github.com/openshift/ruby-hello-world openshift/ruby-22-centos7 hello-world-app
```

Comment:
I'm blind, I've checked the file eariler today. But that was before I've talked with @mfojtik who explained me that only *new* images are in centos namespace. Anyway, thx.